Makefile

```diff
@@ -13,31 +11,13 @@ mdl:
 	mdl --style .markdownlint/style.rb .
 
 build-image:
-	DOCKER_BUILDKIT=1 docker build \
-		--build-arg BUILDKIT_INLINE_CACHE=1 \
-		--cache-from $(REPO_URL):$(BUILD_GO_CACHE_TAG) \
-		--target go-builder \
-		--tag $(IMAGE_NAME):$(BUILD_GO_CACHE_TAG) \
-		.
+	TAG=$(BUILD_TAG) docker buildx bake
```

Comment:
Looks like a nice feature 🤔

Reply:
There's a bit more text in the `docker-bake.hcl` file than the shell commands it replaced, but I like the separation of concerns.